modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,75 @@
+# Candidate Parameters
+
+## Purpose
+
+The Candidate Parameters module allows users to input 
+candidate data that is not timepoint specific.
+
+## Intended Users
+
+The primary types of users are:
+1. Data entry personnel
+2. Site coordinators who are validating the data
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
+collected during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Either one
+is sufficient to access the module.
+
+ - `candidate_parameter_view`
+    - This permission gives the user access to view the module and all of its tabs.
+
+ - `candidate_parameter_edit`
+    - This permission gives the user access to edit the module and all of its tabs.
+
+## Configurations
+
+The following configurations affect the usage of the Candidate Parameters module:
+
+ - `useProband` 
+    - This setting displays/hides the _Proband_ tab of the module.
+
+ - `useFamilyID`
+    - This setting displays/hides the _Family_ tab of the module.
+
+ - `useConsent`
+    - This setting displays/hides the _Consent Status_ tab of the module.
+
+The _Consent Status_ tab dynamically displays consents defined under the 
+`<ConsentModule> <Consent>` tags in the config.xml file. 
```

Comment:
This section is worded a little confusingly, maybe just adding something like "see setup.md for an example" would help